### DISCUSSION TITLE
Reusable q-menu requiring 2 clicks to change :target (on  StackBlitz)

### DISCUSSION THREAD

**Discussion by w-nema** [quasarframework/quasar#16248] [General - Components / Directives / etc] [answered]
### What happened?

I'm trying to use one q-menu for questions and another for answers hoping to change the :target via a context-menu button at each question or answer. The problem is that it takes two clicks for the q-menu to appear.

### What did you expect to happen?

I tried a nextTick workaround which works only when the target is changed (not when the same context menu button is clicked more than once in a row)

### Reproduction URL

https://stackblitz.com/edit/quasarframework-jzubsn

### How to reproduce?

1. Click the "REUSABLE Q-MENU" to open the example tab
2. Click any context menu vertical-dot button on the right side, it shows the q-menu
3. Click it again, it no longer shows the q-menu
4. Click a different context menu button, then go back and click the previous one, it works

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

### Quasar info output

```Shell
Operating System - Windows_NT(10.0.19045) - win32/x64
NodeJs - 18.17.0

Global packages
  NPM - 9.6.7
  yarn - Not installed
  @quasar/cli - 2.2.1
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.12.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.4.5 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.16.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.3.4 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.2.4
  pinia - 2.1.6 -- Intuitive, type safe and flexible Store for Vue
  vuex - Not installed
  vite - 2.9.16 -- Native-ESM powered web dev build tool
  eslint - Not installed
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - LAPTOP-MKANSBQT
  Wi-Fi - 192.168.1.151
```

### Additional context

I tried this, which makes it work on the first time but not next times when pressed in a row
```
  open_aMenu = function (aMenuTargetElId) {
    aMenuTargetEl.value = aMenuTargetElId;
     nextTick(() => {
       aMenuRef.value.show()
    })
  }
```

**Comment by metalsadman** [comment, marked as answer]
https://stackblitz.com/edit/quasarframework-fv9axc?file=src%2Fpages%2FReusableQmenu.vue
